index.d.ts

```diff
@@ -16,6 +20,46 @@ export interface Options {
 	readonly stopOnError?: boolean;
 }
 
+export interface OngoingMappingsStopOptions {
+	/**
+	Whether or not to remove all holes from the result array (caused by pending mappings).
+
+	@default false
```

Comment:
Any thoughts on making this `true` by default?

Reply:
That's fine by me. I don't have any strong opinion. Personally, I will probably set this option explicitly whenever I use `.stop`, regardless of what is the default. I'll change it.

Reply:
@Richienb do you have any opinion on what would be the best default value here?

Reply:
Lets change it to `true` then.

Reply:
> Lets change it to `true` then.

Agree, to match its behaviour with `pMapSkip` which could have left holes but didn't.